### PR TITLE
[FIX] Add amenities tip to tour and fix tips for taxonomy and menu.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

# Steps to review:
- [ ] login as admin
- [ ] go to /admin/structure/taxonomy and click on start Tour
- [ ] verify you can see tip for amenities also on tip 9 operetion list open
- [ ] got to /admin/structure/menu
- [ ] verify in the tip 7 operetion list open
- [ ] go to /admin/structure/taxonomy/manage/color/add
- [ ] verify on tip 5 you can see window for select color

### CODE REVIEW

**Comment by AlexNetman** [comment]
retest this please

**Comment by ivan-berezhnov** [comment]
retest this please

**Comment by ivan-berezhnov** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2066 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2066 |
| Installation process | http://install.openy-dev.ffwua.com/build2066/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2066/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build2066

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2066/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2066/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2066/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2066/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2066/blogWCAG2AAhtmlcs.txt

File: modules/openy_features/openy_menu/openy_menu.install [hunk 1]
```diff
@@ -15,9 +15,17 @@ function openy_menu_update_8001() {
...
  */
 function openy_menu_update_8002() {
   openy_menu_update_8001();
 }
+
+/**
+ * Update menu tutorials.
+ */
+function openy_menu_update_8003() {
+	openy_menu_update_8001();
```
Comment (alexschedrov) [inline modules/openy_features/openy_menu/openy_menu.install]: It's better to copy & paste code that triggering updates in updates. The same for update above.

File: modules/openy_features/openy_taxonomy/openy_taxonomy.install [hunk 2]
```diff
@@ -38,3 +38,10 @@ function openy_taxonomy_update_8002() {
     }
   }
 }
+
+/**
+ * Update tour with token support for demo.
+ */
+function openy_taxonomy_update_8003() {
+	openy_taxonomy_update_8002();
```
Comment (alexschedrov) [inline modules/openy_features/openy_taxonomy/openy_taxonomy.install]: Same here. Plus you have 4 spaces, instead of 2.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

File: modules/openy_features/openy_menu/openy_menu.install (hunk 1 above)
Comment (ivan-berezhnov) [reply modules/openy_features/openy_menu/openy_menu.install]: Update number two has already been completed and I do not need to change it then, right?
As I understood for update number three do 
function openy_menu_update_8003() {
	openy_menu_update_8002();
}
or you mean copy code from first update?

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2066/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2068 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2068 |
| Installation process | http://install.openy-dev.ffwua.com/build2068/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2068/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build2068

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2068/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2068/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2068/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2068/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2068/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2068/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2069 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2069 |
| Installation process | http://install.openy-dev.ffwua.com/build2069/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2069/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build2069

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2069/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2069/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2069/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2069/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2069/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2070 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2070 |
| Installation process | http://install.openy-dev.ffwua.com/build2070/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2070/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build2070

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2070/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2070/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2070/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2070/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2070/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2069/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2071 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2071 |
| Installation process | http://install.openy-dev.ffwua.com/build2071/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2071/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build2071

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2071/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2071/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2071/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2071/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2071/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2070/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build2071/profiles/contrib/openy/build/reports/behat

**Review by AlexNetman — APPROVED** [review]

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build2075 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build2075 |
| Installation process | http://install.openy-dev.ffwua.com/build2075/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/2075/

**Comment by bjwilkins** [comment]
Decoupling report http://ci.openymca.org/build2075

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build2075/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build2075/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build2075/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build2075/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build2075/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build2075/profiles/contrib/openy/build/reports/behat

**Pull request merged**